client/shaders/extract_bloom/opengl_fragment.glsl

```diff
@@ -23,7 +23,7 @@ void main(void)
 	vec2 uv = varTexCoord.st;
 	vec3 color = texture2D(rendered, uv).rgb;
 	// translate to linear colorspace (approximate)
-	color = pow(clamp(color, 0.0, 1.0), vec3(2.2));
+	color = pow(color, vec3(2.2));
```

Comment:
shoudn't we leave this for GLES where `centroid` is not usable?

Reply:
I'd remove it until we know for sure there is a problem with GLES. This does diminish the bloom effect a little, and always has been a hack.

I suppose the correct way in GLES would we to use these: https://registry.khronos.org/OpenGL-Refpages/es3/html/interpolateAtCentroid.xhtml

Reply:
I can't test with GLES :(

OK... So for this. I'll put the clamp back in, but conditioned on GL_ES.

Reply:
See newest commit.